Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocCompundColumnDefinitionTest.cs

```diff
@@ -60,8 +63,21 @@ public void GetComparer_WithPropertyPathNull ()
 
       var comparers = ((CompoundComparer<BocListRow>)comparer).Comparers;
       Assert.That(comparers.Count, Is.EqualTo(2));
-      Assert.That(((BusinessObjectPropertyPathBasedComparer)comparers[0]).PropertyPath, Is.InstanceOf<NullBusinessObjectPropertyPath>());
+      Assert.That(((BusinessObjectPropertyPathBasedComparer)comparers[0]).PropertyPath,
+          Is.InstanceOf<NullBusinessObjectPropertyPath>());
       Assert.That(((BusinessObjectPropertyPathBasedComparer)comparers[1]).PropertyPath, Is.SameAs(propertyPath1.Object));
     }
+
+    [Test]
+    public void GetValidationFailureMatcher_ReturnsInstanceOfCompoundValidationFailureMatcher ()
+    {
+      var column = new BocCompoundColumnDefinition();
+
+      var validationFailureMatcher = ((IBocColumnDefinitionWithValidationSupport)column).GetValidationFailureMatcher();
+
+      Assert.That(validationFailureMatcher, Is.InstanceOf<CompoundBusinessObjectPropertyPathBasedValidationFailureMatcher>());
+      Assert.That(((CompoundBusinessObjectPropertyPathBasedValidationFailureMatcher)validationFailureMatcher).PropertyPaths,
+          Is.EqualTo(column.PropertyPathBindings.Cast<PropertyPathBinding>().Select(p => p.GetPropertyPath())));
```

Comment:
```suggestion
      Assert.That(
          ((CompoundBusinessObjectPropertyPathBasedValidationFailureMatcher)validationFailureMatcher).PropertyPaths,
          Is.EqualTo(column.PropertyPathBindings.Cast<PropertyPathBinding>().Select(p => p.GetPropertyPath())));
```
There's always symetrical line wraps.